The doc subpkg of tig fails the doc test

The documentation subpackage of tig fails testing with the following error:

```
2025/02/04 22:46:30 INFO Usage: tig        [options] [revs] [--] [paths]
2025/02/04 22:46:30 INFO    or: tig log    [options] [revs] [--] [paths]
2025/02/04 22:46:30 INFO    or: tig show   [options] [revs] [--] [paths]
2025/02/04 22:46:30 INFO    or: tig reflog [options] [revs]
2025/02/04 22:46:30 INFO    or: tig blame  [options] [rev] [--] path
2025/02/04 22:46:30 INFO    or: tig grep   [options] [pattern]
2025/02/04 22:46:30 INFO    or: tig refs   [options]
2025/02/04 22:46:30 INFO    or: tig stash  [options]
2025/02/04 22:46:30 INFO    or: tig status
2025/02/04 22:46:30 INFO    or: tig <      [git command output]
2025/02/04 22:46:30 INFO
2025/02/04 22:46:30 INFO Options:
2025/02/04 22:46:30 INFO   +<number>       Select line <number> in the first view
2025/02/04 22:46:30 INFO   -v, --version   Show version and exit
2025/02/04 22:46:30 INFO   -h, --help      Show help message and exit
2025/02/04 22:46:30 INFO   -C <path>       Start in <path>
2025/02/04 22:46:30 WARN + exit 0
2025/02/04 22:46:30 INFO running test pipeline for subpackage tig-doc
2025/02/04 22:46:30 INFO building test workspace in: '/home/user/tmp/melange-guest-3584100686-tig-doc' with apko
2025/02/04 22:46:30 [DEBUG] GET https://packages.wolfi.dev/os/apk-configuration
2025/02/04 22:46:30 INFO setting apk repositories: [/home/user/src/wolfi-os/packages https://packages.wolfi.dev/os]
2025/02/04 22:46:30 INFO image configuration:
2025/02/04 22:46:30 INFO   contents:
2025/02/04 22:46:30 INFO     build repositories: []
2025/02/04 22:46:30 INFO     runtime repositories: []
2025/02/04 22:46:30 INFO     keyring:      []
2025/02/04 22:46:30 INFO     packages:     [tig-doc apk-tools grep man-db texinfo]
2025/02/04 22:46:30 INFO installing ca-certificates-bundle (20241121-r1)
2025/02/04 22:46:30 INFO installing wolfi-baselayout (20230201-r16)
2025/02/04 22:46:30 INFO installing glibc (2.40-r8)
2025/02/04 22:46:30 INFO installing ld-linux (2.40-r8)
2025/02/04 22:46:30 INFO installing libgcc (14.2.0-r8)
2025/02/04 22:46:30 INFO installing glibc-locale-posix (2.40-r8)
2025/02/04 22:46:30 INFO installing zlib (1.3.1-r5)
2025/02/04 22:46:30 INFO installing libcrypto3 (3.4.0-r6)
2025/02/04 22:46:30 INFO installing libssl3 (3.4.0-r6)
2025/02/04 22:46:30 INFO installing apk-tools (2.14.4-r2)
2025/02/04 22:46:30 INFO installing libpcre2-8-0 (10.44-r2)
2025/02/04 22:46:30 INFO installing grep (3.11-r2)
2025/02/04 22:46:30 INFO installing libpipeline (1.5.8-r0)
2025/02/04 22:46:30 INFO installing libbz2-1 (1.0.8-r10)
2025/02/04 22:46:30 INFO installing libxcrypt (4.4.38-r0)
2025/02/04 22:46:30 INFO installing libcrypt1 (2.40-r8)
2025/02/04 22:46:30 INFO installing perl (5.40.1-r0)
2025/02/04 22:46:30 INFO installing libstdc++ (14.2.0-r8)
2025/02/04 22:46:30 INFO installing groff (1.23.0-r3)
2025/02/04 22:46:30 INFO installing libseccomp (2.6.0-r0)
2025/02/04 22:46:30 INFO installing gdbm (1.24-r2)
2025/02/04 22:46:30 INFO installing man-db (2.13.0-r1)
2025/02/04 22:46:30 INFO installing ncurses-terminfo-base (6.5_p20241228-r0)
2025/02/04 22:46:30 INFO installing ncurses (6.5_p20241228-r0)
2025/02/04 22:46:30 INFO installing texinfo (7.2-r0)
2025/02/04 22:46:31 INFO installing tig-doc (2.5.11-r0)
2025/02/04 22:46:31 INFO installing wolfi-keys (1-r8)
2025/02/04 22:46:31 INFO installing busybox (1.37.0-r0)
2025/02/04 22:46:31 INFO installing wolfi-base (1-r6)
2025/02/04 22:46:31 INFO OCI layer digest: sha256:04a94f84c1bae0ce7bc582b15e64b4d6ede8edd1d5286da582eb7f78f4e6cbb0
2025/02/04 22:46:31 INFO OCI layer diffID: sha256:6e18b227f05353d5a195ca6762d681f6863cd8115d99672369c3042afaa7de76
2025/02/04 22:46:31 INFO saving OCI image locally: apko.local/cache:705e29658b05e7f5dba50ffdae2b7f2f873796476cd738e1b1b397f429facebe
2025/02/04 22:46:32 WARN skipping local domain tagging apko.local/cache:705e29658b05e7f5dba50ffdae2b7f2f873796476cd738e1b1b397f429facebe as index.docker.io/library/melange:latest
2025/02/04 22:46:32 INFO running step "test/docs"
2025/02/04 22:46:32 WARN + '[' -d /home/build ]
2025/02/04 22:46:32 WARN + cd /home/build
2025/02/04 22:46:32 WARN + exit 0
2025/02/04 22:46:32 INFO running step "docs readability check"
2025/02/04 22:46:32 WARN + '[' -d /home/build ]
2025/02/04 22:46:32 WARN + cd /home/build
2025/02/04 22:46:32 WARN + basename /home/build/melange-out/tig-doc
2025/02/04 22:46:32 WARN + doc_pkg=tig-doc
2025/02/04 22:46:33 WARN + apk info -L tig-doc
2025/02/04 22:46:33 WARN + wc -l
2025/02/04 22:46:33 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:46:33 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:46:33 INFO See:
2025/02/04 22:46:33 WARN + '[' 3 -le 3 ]
2025/02/04 22:46:33 WARN + echo See:
2025/02/04 22:46:33 WARN + apk info -L tig-doc
2025/02/04 22:46:33 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:46:33 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:46:33 INFO tig-doc-2.5.11-r0 contains:
2025/02/04 22:46:33 INFO var/lib/db/sbom/tig-doc-2.5.11-r0.spdx.json
2025/02/04 22:46:33 INFO
2025/02/04 22:46:33 WARN + echo 'This package [tig-doc] is completely empty (i.e. installs no files).'
2025/02/04 22:46:33 WARN + echo 'Please check the package build for proper docs installation, and either:'
2025/02/04 22:46:33 WARN + echo '  (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or'
2025/02/04 22:46:33 WARN + echo '  (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or'
2025/02/04 22:46:33 WARN + echo '  (c) remove the docs subpackage entirely'
2025/02/04 22:46:33 WARN + exit 1
2025/02/04 22:46:33 INFO This package [tig-doc] is completely empty (i.e. installs no files).
2025/02/04 22:46:33 INFO Please check the package build for proper docs installation, and either:
2025/02/04 22:46:33 INFO   (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or
2025/02/04 22:46:33 INFO   (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or
2025/02/04 22:46:33 INFO   (c) remove the docs subpackage entirely
2025/02/04 22:46:33 INFO pod 28bab6da2c9517ae10d71a680fffacf10bd68d00461ed76464488e7c3863278e terminated
2025/02/04 22:46:33 INFO pod 93316f3709caec74cc5bfa6ad8fed707cb65d3a1782ce30d2ee95d2095454a06 terminated
2025/02/04 22:46:33 INFO ERROR: failed to test package. the test environment has been preserved:
2025/02/04 22:46:33 INFO   workspace dir: /home/user/tmp/melange-workspace-1035755094
2025/02/04 22:46:33 INFO   guest dir: /home/user/tmp/melange-guest-3584100686
2025/02/04 22:46:33 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:121: test/tig] Error 1
```

--- a/tig.yaml
+++ b/tig.yaml
@@ -2,7 +2,7 @@
 package:
   name: tig
   version: "2.5.11"
-  epoch: 0
+  epoch: 1
   description: Text-mode interface for the git revision control system
   copyright:
     - license: GPL-2.0-or-later
@@ -30,6 +30,8 @@ pipeline:
   - uses: autoconf/make
 
   - uses: autoconf/make-install
+    with:
+      opts: install-doc-man
 
   - uses: strip
 
@@ -38,6 +40,9 @@ subpackages:
     pipeline:
       - uses: split/manpages
     description: tig manpages
+    test:
+      pipeline:
+        - uses: test/docs
 
 update:
   enabled: true

